pypose/optim/optimizer.py

```diff
@@ -175,6 +191,19 @@ def __init__(self, model, solver=None, kernel=None, corrector=None, weight=None,
             self.corrector = Trivial() if corrector is None else corrector
         self.weight = weight
 
+    def normalize_R_weight(self, R, weight=None):
```

Comment:
is this function identical to both GN and LM? if yes, how about moving it to the class of `RobustModel`, so that it only appear once?

Reply:
I also need a meeting with you to explain the math background of this function.